app/assets/javascripts/admin/orders/controllers/bulk_cancel_controller.js.coffee

```diff
@@ -0,0 +1,14 @@
+angular.module("admin.orders").controller "bulkCancelCtrl", ($scope, $http, $timeout) ->
+
+  $scope.cancelOrder = (orderIds, sendEmailCancellation, restock_items) ->
+    $http(
+      method: 'post'
+      url: "/admin/orders/bulk_cancel?order_ids=#{orderIds}&send_cancellation_email=#{sendEmailCancellation}&restock_items=#{restock_items}" ).then(->
```

Comment:
Usually in REST request we use `order_ids[]=1&order_ids[]=25&...` but happy to go with this solution.